build.gradle

```diff
@@ -1493,12 +1493,22 @@ task docs(type: ScalaDoc) {
   })
   destinationDir = file("${rootProject.buildDir}/docs")
 }
+
+task buildSqlFuncDocs(type: Exec) {
+  if (!file("${rootProject.projectDir}/build-artifacts/scala-2.11/snappy/jars").exists()) {
+    dependsOn product
+  }
```

Comment:
@paresh-p11 Its better to remove this condition and always have dependsOn because even if the jars directory exists, it might be out of date

Reply:
done